### DISCUSSION TITLE
Allow us to disable the player sleeping actionbar message.

### DISCUSSION THREAD

**Discussion by RealTriassic** [PurpurMC/Purpur#465] [__Completed__] [answered]
Allow us to disable the message from appearing in the actionbar every time a player sleeps in a bed. This message is part of the Vanilla game and looks something like "1/30 players sleeping" and it appears in the actionbar, even if can-skip-night is set to false in purpur.yml, I'd love to see this message being toggle-able in purpur.yml, thanks!

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1246 https://github.com/pl3xgaming/Purpur/commit/3cb81112e2b850ab8372643527b71ee585af817d

You can disable the messages by supplying an empty string

```yaml
sleep-skipping-night: ""
sleeping-players-percent: ""
```

Or customize them

```yaml
sleep-skipping-night: "Sleeping through this night"
sleeping-players-percent: "<count>/<total> players sleeping"
```

Leave them as default to let the client's use their own translatable components

```yaml
sleep-skipping-night: "default"
sleeping-players-percent: "default"
```
